DISCOVER/_QA_Protocols.md

```diff
@@ -0,0 +1,17 @@
+## Improving the Q&A Process
+
+### Inclusive Question Submission
+To make the Q&A sessions after talks more inclusive and accessible, it's essential to provide multiple ways for attendees to submit their questions. This will benefit not only hearing/speech impaired individuals but also quieter groups, such as younger participants or those who may be shy or less confident.
+
+### Written Format Options
+- **Website Form:** Create an online form on the event's website where attendees can submit their questions before or during the talk.
+- **Twitter Hashtag:** Designate a specific Twitter hashtag for the event or session where attendees can post their questions. This also helps in engaging a wider audience on social media.
+- **Text Messaging:** Provide a phone number for attendees to text their questions during the talk.
+- **Question Cards:** Distribute question cards that attendees can fill out and hand to event staff during the session.
```

Comment:
Many conferences also use a system for this, such as Mentimeter. These systems have the advantage that the audience can see the questions that others have asked, and the audience can vote on questions, so that the questions can be answered that are of most interest to the whole audience.

Reply:
[Slido](https://www.slido.com/) is another app that can be used for this purpose